### PR TITLE
[slide-panel] Fixed slide panel crashing when open by default

### PR DESCRIPTION
**Pull request by sdadn**
### Summary

This PR fixes a defect where `terra-slide-panel` crashes if it is open by upon page load.

### Deployment Link

https://terra-framework-deployed-pr-#.herokuapp.com/

### Testing

An additional WDIO test has been added to test this edge case.

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Review by sycombs — APPROVED** [review]

**Review by JessieRandle — APPROVED** [review]

File: packages/terra-slide-panel/src/SlidePanel.jsx [hunk 1]
```diff
@@ -84,9 +84,17 @@ class SlidePanel extends React.Component {
...
       this.setDisclosingNode(this.lastClicked);
       this.panelNode.focus();
-    } else if (!this.props.isOpen && this.props.isOpen !== prevProps.isOpen) {
-      // Return focus to the disclosing element
-      this.disclosingNode.focus();
+      return;
+    }
+
+    if (!this.props.isOpen && this.props.isOpen !== prevProps.isOpen) {
+      if (this.disclosingNode !== undefined) {
```
Comment (smason0) [inline packages/terra-slide-panel/src/SlidePanel.jsx:91]: Should this also check for `null`? Or just a falsy check: `if (this.disclosingNode)`

Comment (sdadn) [reply packages/terra-slide-panel/src/SlidePanel.jsx:91]: I can add a check for `null. Good catch.

Comment (sdadn) [reply packages/terra-slide-panel/src/SlidePanel.jsx]: Updated here: 4e32267ead59031f52336a8cfda6ecd143bb7c19

**Review by smason0 — APPROVED** [review]

**Review by sycombs — APPROVED** [review]

**Review by trandrew1023 — APPROVED** [review]
